components/eam/src/chemistry/modal_aero/modal_aero_convproc.F90

```diff
@@ -2243,7 +2242,7 @@ subroutine ma_convproc_tend(                                           &
             cdt(k) = 0.0_r8
             if ((icwmr(icol,k) > clw_cut) .and. (rprd(icol,k) > 0.0)) then 
 !              if (iconvtype == 1) then
-                  tmpf = fa_u(k) !0.5_r8*cldfrac_i(k)
+               tmpf = fa_u(k) !0.5_r8*cldfrac_i(k)
                   cdt(k) = (tmpf*dp(i,k)/mu_p_eudp(k)) * rprd(icol,k) / &
```

Comment:
Would you align the indenting for this line so that it matches the previous line?